packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -412,20 +411,32 @@ export function _addResourceSpans(
     return;
   }
 
+  const parsedUrl = parseUrl(resourceUrl);
+
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const data: Record<string, any> = {};
   setResourceEntrySizeData(data, entry, 'transferSize', 'http.response_transfer_size');
   setResourceEntrySizeData(data, entry, 'encodedBodySize', 'http.response_content_length');
   setResourceEntrySizeData(data, entry, 'decodedBodySize', 'http.decoded_response_content_length');
+
   if ('renderBlockingStatus' in entry) {
     data['resource.render_blocking_status'] = entry.renderBlockingStatus;
   }
+  if (parsedUrl.protocol) {
+    data['url.scheme'] = parsedUrl.protocol;
+  }
+
+  if (parsedUrl.host) {
+    data['server.address'] = parsedUrl.host;
+  }
+
+  data['url.same_origin'] = resourceUrl.includes(WINDOW.location.origin);
```

Comment:
Alternatively we can check if Window.location.host === parsedUrl.host? But I think this might be a bit more robust as it includes the host + scheme?

Reply:
This breaks if someone has the URL in query parameters - we should directly check host.

Reply:
👍